FNAC.js

```diff
@@ -0,0 +1,316 @@
+{
+	"translatorID": "825f208c-eb96-4372-9d21-b879c3a910bb",
+	"label": "FNAC",
+	"creator": "César Lizurey",
+	"target": "^https?://(www\\.)?fnac\\.com",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2024-10-21 19:30:59"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2024 César Lizurey
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	// Check if the URL matches the desired format
+	var match = url.match(/^https:\/\/www\.fnac\.com\/a(\d+)\/([\w-]+)$/);
+
+	if (match) {
+		Z.debug("URL OK");
+
+		// Get the second <a> element in the breadcrumb to check the itemType
+		var secondLink = doc.querySelector('nav.f-breadcrumb > ul.f-breadcrumb__list > li:nth-of-type(2)');
```

Comment:
> This seems way too likely to change - how about `nav > ul > :nth-child(2)`? Still not great, but the less reliance we have on presentation class names, the better.

Actually I wanted to prevent bugs in case of a the apparition of a new "nav" element in the page. The 2 classes have been very consistent for a long time.